### ISSUE TITLE
Build fails when used with HTML <a> tag

### ISSUE THREAD

**Issue by renjithk** [LikeTheSalad/android-stem#103]
I have added android-stem library 2.10.0 as follows

```
plugins {
 id "com.likethesalad.stem" version "2.10.0"
}
```

Here is a string value where the placeholder is added.

```
<string name="help_text_with_link">
     For more help on how to use the ${app_name}, please visit <a href="https://somedomain.com/help/">this site</a>
</string>
```

However, it fails to build with the following error.
![image](https://github.com/user-attachments/assets/1fb19942-fae1-4df6-b45f-cc7b320fdb8a)

Is HTML `<a>` tag not supported on 2.10.0?

**Comment by LikeTheSalad** [comment]
Hi @renjithk 

Thank you for the feedback. I'll take a look later this week.

**Comment by LikeTheSalad** [comment]
I've just released version `2.10.1` with a fix, please feel free to reopen this issue if the problem persists with the new version.

Thank you for your feedback.
